docs/tutorials/analysis/3D/event_sampling.ipynb

```diff
@@ -747,8 +746,11 @@
    "outputs": [],
    "source": [
     "%%time\n",
+    "n_obs = len(tstarts)\n",
+    "irf_paths = [Path(irf_filename)] * n_obs\n",
+    "events_paths = []\n",
     "for idx, tstart in enumerate(tstarts):\n",
-    "\n",
+    "    irf_path = load_cta_irfs(irf_paths[idx])\n",
```

Comment:
Is it a path or a `dict`? I would assume the later

Reply:
a dict yes